torchbox/torchbox.com
# Allow only distinct blog posts

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1409752818)

### Description of Changes Made
- Implement fix to prevent duplicated blog posts being shown on a BlogIndexPage
- I've also confirmed that this is not necessary on WorkIndexPages

### How to Test
Confirm that no duplicate blog posts are present when filtering is applied - you could also include a print statement or logging to confirm no duplicates are present 


### Screenshots
Without Fix
![image](https://github.com/torchbox/torchbox.com/assets/68611528/bceed652-e484-42fb-80c2-13b29bc82f80)


With Fix
![image](https://github.com/torchbox/torchbox.com/assets/68611528/129fd93e-e0ad-4bb7-9d0b-cecd578762cf)

<details>
  <summary>Expand to see more</summary>

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
